src/Animations/Animations/Shared/Animations/StoryBoard.cs

```diff
@@ -27,7 +27,9 @@ public List<AnimationBase> Animations
             foreach (var animation in Animations)
             {
                 if (animation.Target == null)
+                {
```

Comment:
Is this a general rule or can it be specified for specific cases? I don't mind dropping braces in case of exception / return. Adding in this particular case is ok for me.

Reply:
Yes => in the file Toolkit.ruleset